proposals/libocispec.md

```diff
@@ -0,0 +1,37 @@
+# OCI libocispec project proposal
+
+## Abstract
+Provide a C library for parsing and handle OCI runtime spec file and OCI image spec files.
```

Comment:
“handle” → “handling”.  Or maybe “representing”?  Or maybe “passing”?  The library seems to handle parsing from various formats into C types and manipulating those C types in memory.  There's nothing about implementing the OCI spec semantics.  It would be good to have a short word/phrase here to make that distinction clear, and I don't think “handling” is specific enough.

Also, the input may be in a file, or may just be in a string (e.g. `oci_container_parse_data`).  What they have in common is that they all parse JSON.  So I'd rather end with “OCI Runtime and Image specification JSON.” or some such.